Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -258,9 +263,9 @@ lineWidth : expression;
 // and let the resolver handle it.
 debugPrintStmt : debugPrint (whiteSpace outputList)?;
 // We split it up into separate rules so that we have context classes generated that can be used in declarations/references.
-debugPrint : debugModule whiteSpace? DOT whiteSpace? debugPrintSub;
+debugPrint : debugModule whiteSpace? DOT whiteSpace? printMethod;
 debugModule : DEBUG;
-debugPrintSub : PRINT;
+printMethod : PRINT;
```

Comment:
I can't help but wonder if it should be `PRINT outputList`? 

In fact, I wonder whether we really should have `debugPrint` to begin if the behavior of `Print` is universal regardless whether it's on a `Debug` pseudo object or some other objects.

Reply:
No, we need a wrapper around the `PRINT` token alone becasue in the resolver everything works off subclasses of `ParserRuleContext` and not `IParseTree`, I guess for historical reasons. Tokens are no `PaserRuleContext`s. Thus, we need a wrapper to allow passing it into a `MemberAccessDefaultBinding` in `DefaultBindingContext` in the reference resolver.

We also need `debugPrint` because we resolve the `Debug` part differently in the `IdentifierReferenceResolver`.

Reply:
Thinking about it again, I think we can get rid of the special resolution for `Debug`. I will test wether I can do that.

Reply:
I now understand why we have a rule for `PRINT` alone; but we only have this:
```
debugPrintStmt : debugPrint (whiteSpace outputList)?;
```

which probably needs to be redefined into:
```
printStmt: printMethod (whiteSpace outputList)?;
```

to make the `Print` method parse the same way regardless of what object it is on.

Reply:
No, I just tested; `debugPrintStmt` will disappear entirely.